desc/coils.py

```diff
@@ -1369,25 +1383,37 @@ def _compute_position(self, params=None, grid=None, **kwargs):
         grid : Grid or int, optional
             Grid of coordinates to evaluate at. Defaults to a Linear grid.
             If an integer, uses that many equally spaced points.
+        dx1 : bool
+            If True, also return dx/ds for each curve.
 
         Returns
         -------
         x : ndarray, shape(len(self),source_grid.num_nodes,3)
             Coil positions, in [R,phi,Z] or [X,Y,Z] coordinates.
+        x_s : ndarray, shape(len(self),source_grid.num_nodes,3)
+            Coil position derivatives, in [R,phi,Z] or [X,Y,Z] coordinates.
+            Only returned if dx1=True.
 
         """
         basis = kwargs.pop("basis", "xyz")
+        keys = ["x"]
+        if dx1:
+            keys += ["x_s"]
```

Comment:
Could consolidate this into a single line like you did above:
`keys = ["x", "x_s"] if dx1 else ["x"]`